pkg/machine/e2e/README.md

```diff
@@ -12,8 +12,15 @@
 ### HyperV
 
 1. Open a powershell as admin
-2. $env:CONTAINERS_MACHINE_PROVIDER="hyperv"
-3. $env:MACHINE_IMAGE="https://fedorapeople.org/groups/podman/testing/hyperv/fedora-coreos-38.20230830.dev.0-hyperv.x86_64.vhdx.zip"
-4. `./test/tools/build/ginkgo.exe -vv  --tags "remote exclude_graphdriver_btrfs btrfs_noversion exclude_graphdriver_devicemapper containers_image_openpgp remote" -timeout=90m --trace --no-color  pkg/machine/e2e/. `
+1. $env:CONTAINERS_MACHINE_PROVIDER="hyperv"
+1. $env:MACHINE_IMAGE="https://fedorapeople.org/groups/podman/testing/hyperv/fedora-coreos-38.20230830.dev.0-hyperv.x86_64.vhdx.zip"
+1. `./test/tools/build/ginkgo.exe -vv  --tags "remote exclude_graphdriver_btrfs btrfs_noversion exclude_graphdriver_devicemapper containers_image_openpgp remote" -timeout=90m --trace --no-color  pkg/machine/e2e/. `
+
+Note: Add `--focus-file "basic_test.go" ` to only run basic test
+
+### WSL
+1. Open a powershell as a regular user
+1. Build and copy win-sshproxy into bin/
+1. `./test/tools/build/ginkgo.exe -vv  --tags "remote exclude_graphdriver_btrfs btrfs_noversion exclude_graphdriver_devicemapper containers_image_openpgp remote" -timeout=90m --trace --no-color  pkg/machine/e2e/. `

```

Comment:
A make target would be nice.
`make test-machine` or `make test-machine-wsl`?

Reply:
see #20028